pkg/scheduler/plugins/nodenumaresource/plugin.go

```diff
@@ -132,8 +133,12 @@ func (p *Plugin) PreFilter(ctx context.Context, cycleState *framework.CycleState
 	qosClass := extension.GetPodQoSClass(pod)
 	priorityClass := extension.GetPriorityClass(pod)
 	if (qosClass == extension.QoSLSE || qosClass == extension.QoSLSR) && priorityClass == extension.PriorityProd {
-		if resourceSpec.PreferredCPUBindPolicy == schedulingconfig.CPUBindPolicyFullPCPUs ||
-			resourceSpec.PreferredCPUBindPolicy == schedulingconfig.CPUBindPolicySpreadByPCPUs {
+		preferredCPUBindPolicy := resourceSpec.PreferredCPUBindPolicy
+		if preferredCPUBindPolicy == "" || preferredCPUBindPolicy == schedulingconfig.CPUBindPolicyNone {
```

Comment:
is it possible to specify bind no CPUs?

Reply:
LSE/LSE Pods must specify CPUBindPolicy, which cannot be set to None.

Reply:
can we remove the `CPUBindPolicyNone` option

Reply:
hmm..It's better to rename `CPUBindPolicyNone` to `CPUBindPolicyDefault`. WDYT?

Reply:
totally agree